src/client/NetworkingClient.cpp

```diff
@@ -127,7 +127,7 @@ void NetworkingClient::RealtimeClientSend() {
         if (mAcceptingDirectionRead) {
             sf::Packet packet;
             sf::Uint32 move_cmd = (sf::Uint32)RealtimeCommand::move;
-            if (packet << mClientId << move_cmd << mCurrentTick << mDirection.x_dir << mDirection.y_dir) {
+            if (packet << mClientId << move_cmd << mCurrentTick << mDirection.x << mDirection.y) {
```

Comment:
this should probably have something like network_game_object right? serializedGameMove?

Reply:
Might be a rare case where inheritance is actually useful.

Reply:
fair enough :) probably something to do down the line, but not a big deal